src/en/news/blog/2022/improving-dashboard-accessibility-for-blind-and-visually-impaired-ceph-users/index.md

```diff
@@ -0,0 +1,134 @@
+
+## Project
+
+For the past few months we have been working on updating the dashboard to meet WCAG 2.1 level A conformance so that blind and visually impaired Ceph users can also use it comfortably to manage and monitor their Ceph clusters.
+
+To achieve this goal, we had to:
+
+1. Test and identify areas of the dashboard in need of an accessibility update
+2. Apply fixes based on WCAG 2.1 guidelines
+3. Add automated testing to prevent regressions
+
+### Identifying accessibility issues
+
+After testing several recommended [accessibilty checkers](https://www.w3.org/WAI/ER/tools/?q=wcag-21-w3c-web-content-accessibility-guidelines-21) by the W3C, we chose the [accessibility insights plugin](https://accessibilityinsights.io/docs/en/web/overview/) to help quickly identify the most common accessibility issues for each page of the the dashboard.
+
+### Accessibility improvements
+
+Based on the identified issues, we improved the accessibility of several key components including:
+
+#### Login Page
+
+By improving the color contrast of the login form, input fields users can now clearly read the labels and use the keyboard to navigate the inputs.
+
+![login page](images/login.png)
+
+#### Landing Page
+
+The Dashboard page displays key monitoring information about cluster status, capacity and performance. We improved on the following aspects:
+
+- updated the cluster labels (HEALTH_WARN, HEALTH_OK, HEALTH_ERROR) to be more descriptive (Warning, Ok, Error)
+
+- Used darker colors for labels to make them more legible
+- Added accessible names to informative icons for screen readers
+
+![Dashboard page](images/dashboard.png)
+
+#### Navigation
+
+The navigation component enables quick access to all the Dashboard's sections. I was able to make the following accessibility updates:
+
+- Add accessible roles for all dropdown menus for easier detection by screen readers
+- Used darker background colors and increased label font size for better legibility
+
+![Navigation](images/navigation.png)
+
+#### Datatables
+
+The datatable component is widely used in the Dashboard to display more detailed information. The improvements made here include:
+
+- Added accessible labels for row selection checkboxes for better identification by screen readers
+- Added a custom pagination component with better UX and accessibility
+- Changed table action button label colors from red to cyan for better legibility and focus
+
+![Datatable](images/datatable.png)
+
+Many more sub-components were also updated and can be found on [GitHub](https://github.com/ceph/ceph/pulls/nsedrickm)
+
+### Accessibility testing
+
+To prevent future regressions, we added end to end accessibility tests for high impact components like the dashboard and navigation. If any future regressions on these components occurs, a tabulated error description is generated and added to the Cypress log for debugging.
```

Comment:
I assume you mean the "health page" here?
```suggestion
To prevent future regressions, we added end to end accessibility tests for high impact components like the health page and navigation. If any future regressions on these components occurs, a tabulated error description is generated and added to the Cypress log for debugging.
```